src/www/app/outline_server_repository/shadowsocks_session_config.ts

```diff
@@ -13,10 +13,10 @@
 // limitations under the License.
 
 // Represents a Shadowsocks proxy server configuration.
-export interface ShadowsocksConfig {
+export interface ShadowsocksSessionConfig {
```

Comment:
This doesn't belong in the repository, as it's the interface for the tunnel. If anything, it should be defined in `tunnel.ts`.

Reply:
Ahhh that makes so much more sense!